### DISCUSSION TITLE
An error will occur if build with Vercel.

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#1171] [Questions] [answered]
Only me?
![スクリーンショット 2021-04-16 14 59 05](https://user-images.githubusercontent.com/42270511/114978547-4bf41c80-9ec4-11eb-99fe-4c935276b4f0.png)

```
 > Build error occurred
Error: supabaseKey is required.
at new SupabaseClient (/vercel/workpath0/node_modules/@supabase/supabase-js/dist/main/SupabaseClient.js:48:19) at createClient (/vercel/workpath0/node_modules/@supabase/supabase-js/dist/main/index.js:25:12)
```

My initSupabase.tsx
```
import { createClient } from "@supabase/supabase-js";

const SUPABASE_URL = "https://xxxx.supabase.co";
const SUPABASE_KEY = process.env.NEXT_PUBLIC_SUPABASE_KEY || "";

export const supabase = createClient(SUPABASE_URL, SUPABASE_KEY);
```

My environment.
--Next.js
--Typescript
--Vercel

**Comment by toddgeist** [comment, marked as answer]
did you add the Env Variables in Vercel?

https://vercel.com/docs/environment-variables

**Reply by ru88s** [reply]
Thanks to you, I was able to notice.
The env.production.local file was not uploaded to the production environment.
It's my mistake.
Thank you very much.
